package.json

```diff
@@ -6,11 +6,18 @@
   "scripts": {
     "build": "react-scripts build",
     "build-css": "node-sass-chokidar src/ -o src/devBuild",
-    "sass-lint": "sass-lint ./src --config .sass-lint.yml 'src/**/*.scss' --verbose --ignore 'node_modules/**/*'",
+    "sass-lint":
+      "sass-lint ./src --config .sass-lint.yml 'src/**/*.scss' --verbose --ignore 'node_modules/**/*'",
     "server": "nodemon server/app.js",
-    "start": "concurrently --prefix \"[AAF]\" \"npm run watch-css\" \"npm run server\" \"react-scripts start\"",
-    "test": "eslint ./ && npm run sass-lint && react-scripts test --env=jsdom",
-    "watch-css": "npm run build-css && node-sass-chokidar src/ -o src/devBuild/ --watch --recursive"
+    "start":
+      "concurrently --prefix \"[AAF]\" \"npm run watch-css\" \"npm run server\" \"react-scripts start\"",
+    "test:lint": "eslint ./",
+    "test:react": "react-scripts test --env=jsdom",
+    "test:api": "jest Api --env=jsdom",
+    "test":
```

Comment:
Should test continue to watch after it completes, or break after one test cycle?

Reply:
it *should* just stop. Which I think travis does... but locally it just keeps watching and it drives me nuts

Reply:
Oh okay, Yeah it wasn't stopping locally. Lol. But it also doesn't re-run when i make a change.

Reply:
yeah dude, for some reason, the `npm test` command gets stuck locally when running. I recommend using `npm run test:api` and that works without issue. the problem is that the react run is what keeps the watch happening. I don't know why it does it, but that's the issue with using wrappers :|

Reply:
@jwu910 i fixed it using the --forceExit option, but it seems that this is like a workaround for bad code haha.
"Force Jest to exit after all tests have completed running. This is useful when resources set up by test code cannot be adequately cleaned up. Note: This feature is an escape-hatch. If Jest doesn't exit at the end of a test run, it means external resources are still being held on to or timers are still pending in your code. It is advised to tear down external resources after each test to make sure Jest can shut down cleanly."
Source: https://facebook.github.io/jest/docs/en/cli.html#forceexit